accel/tcg/log_instr.c

```diff
@@ -1036,6 +1051,319 @@ void qemu_log_instr_extra(CPUArchState *env, const char *msg, ...)
     va_end(va);
 }
 
+// A printf that takes an array of argments unioned of all possible argument
+// types. Because we cant edit a VA_LIST, and we don't want the exponential blow
+// up of handling all combinations of types, we will bounce individual string
+// sections split in the fmt string to another buffer, then switch on all
+// possible types.
+static void g_string_append_printf_union_args(GString *string, const char *fmt,
+                                              qemu_log_arg_t *args)
+{
+
+// So Clang will not complain about the non-literal format.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-nonliteral"
+#pragma clang diagnostic ignored "-Wformat-security"
+    char bounce_buf[256];
+
+    size_t i = 0;
+    bool format = false;
+    char c;
+    bool is_short, is_long, is_long_long;
+    while ((c = bounce_buf[i++] = *fmt++)) {
+        if (!format) {
+            format = c == '%';
+            is_short = is_long = is_long_long = false;
+        } else {
```

Comment:
```suggestion
        if (!format) {
            format = c == '%';
            is_short = is_long = is_long_long = false;
            continue;
        }
```
lets you deindent the whole thing

Reply:
Done for this and for the similar loop below.